phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/MainController.java

```diff
@@ -64,26 +77,28 @@ public MainController(String applicationName,
                 themeManager,
                 javaFxSettingsManager);
 
+        this.themeManager = themeManager;
         this.javaFxSettingsManager = javaFxSettingsManager;
 
-        installationsController.setOnInstallationAdded(mainWindow::showInstallations);
-
-        appsController.setOnAppLoaded(() -> {
-            containersController.loadContainers();
+        repositoryManager.addCallbacks(this::setDefaultCategoryIcons, e -> {
         });
+
+        installationsController.setOnInstallationAdded(this.mainWindow::showInstallations);
+
+        appsController.setOnAppLoaded(containersController::loadContainers);
     }
 
     public void show() {
-        mainWindow.show();
+        this.mainWindow.show();
     }
 
     public void setOnClose(Runnable onClose) {
         this.mainWindow.setOnCloseRequest(event -> {
             Alert alert = new Alert(Alert.AlertType.CONFIRMATION);
```

Comment:
This looks like a use case for `ConfirmDialog`

Reply:
Can you open a new issue for this?

Reply:
#1623